sdk/daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/TypeSig.scala

```diff
@@ -117,9 +90,9 @@ object TypeSig {
     final case class Var(name: Ref.Name) extends SerializableType
   }
 
-  final case class RecordSig(params: Seq[Ref.Name], fields: Seq[(Ref.Name, SerializableType)])
+  final case class RecordSig(params: Seq[Ref.Name], fields: Map[Ref.Name, SerializableType])
```

Comment:
@remyhaemmerle-da Any reason for it not to be Map?

Reply:
Fine with me.

Reply:
in fact VectorMap is problably better, as you can know the order of the field.
```suggestion
  final case class RecordSig(params: Seq[Ref.Name], fields: VectorMap[Ref.Name, SerializableType])
```

Reply:
Good point. Changed